include/nbl/asset/IMeshPackerV2.h

```diff
@@ -238,6 +273,70 @@ class IMeshPackerV2 : public IMeshPacker<MeshBufferType, MDIStructType>, public
     inline uint32_t getIntBufferBindingsCnt() { return m_virtualAttribConfig.intArrayElementsCnt; }
     inline uint32_t getUintBufferBindingsCnt() { return m_virtualAttribConfig.uintArrayElementsCnt; }
```

Comment:
I'd make a `const VirtualAttribConfig& getVirtualAttribConfig() const` and give up these redundant wrappers

Reply:
nvm `VirtualAttribConfig` is protected